src/index.js

```diff
@@ -480,6 +480,9 @@ function ncc (
             keep_classnames: true,
             keep_fnames: true
           },
+          format: {
+            comments: false
+          },
```

Comment:
I'm not sure we want to do this. The docs say:

> by default it keeps JSDoc-style comments that contain "@license", "@preserve" or start with `!`

Some code relies on comments (I think Angular does func.toString() for example) so if someone wanted it to be preserved we should probably preserve it.

Reply:
While I think code that relies on `function.toString` is fundamentally flawed, I guess a option could be introduced on whether to retain such comments.

Or more maybe even more generally, the minifier options could be exposed on the [`Node.js API`](https://github.com/vercel/ncc#programmatically-from-nodejs), e.g. property `minfierOptions` that is then deep-merged into the static configuration.

Reply:
I would rather not expose terser options since we might replace it with something else.

Are you seeing big differences in bundle sizes with this change?

I noticed you didn't add a test.

Reply:
It saves a few kilobytes, but it's nothing major in most cases.